gnome-extra/gnome-logs/gnome-logs-43.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="Log viewer for the systemd journal"
+HOMEPAGE="https://wiki.gnome.org/Apps/Logs"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+
+RDEPEND="
+	gnome-base/gsettings-desktop-schemas
+	>=dev-libs/glib-2.43.90:2
+	>=gui-libs/gtk-4.6.0:4
+	>=gui-libs/libadwaita-1.2.0
```

Comment:
The diff between the ebuilds is

````
-       >=x11-libs/gtk+-3.22:3
-       >=gui-libs/libhandy-1.5.0:1
+       >=gui-libs/gtk-4.6.0:4
+       >=gui-libs/libadwaita-1.2.0
````

Just FYI, we should leave the `:1` slot requirement in place for libadwaita.